pkgs/development/python-modules/flask-basicauth/default.nix

```
@@ -0,0 +1,35 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, fetchpatch
+, flask
+, lib
+}:
```

We usually put `lib` first, and I've ordered the rest in the order that they're used in the derivation.

```suggestion
{ lib
, buildPythonPackage
, fetchFromGitHub
, fetchpatch
, flask
}:
```